docs/versioned_docs/version-0.x/intro.mdx

```diff
@@ -20,6 +20,8 @@ Application developers can utilize Xanthic to easily switch between different [b
 
 ## Example Usage
 
+Create a cache (with `String` keys and `Integer` values) that can hold up to 2048 elements, where entries expire 5 minutes following their latest access, and removed entries notify a specific event listener:
```

Comment:
```suggestion
Create a cache (with `String` keys and `Integer` values) that can hold up to 2048 elements, where entries expire 5 minutes following their latest access, and removed entries notify a specific event listener, using the default [provider](/provider/) on the classpath:
```

Should this be added or is it too complicated for the homepage

Reply:
Maybe we could add a second paragraph where we explain this a little different, for example

> - you are required to add at least one `cache-provider-x` module to your project
> - if you only added a single cache provider than this will be the default
> - if you add more than one cache provider you will need to set the default using this one-liner: ...
> - if you add more than one cache provider and did not set a default, the order to determinate the default is documented on the provider docs page - it's not recommendet to rely on the order!